ui/src/app/resources/applications/application-details.component.ts

```diff
@@ -34,6 +35,10 @@ export class ApplicationComponent implements OnInit {
         },
         err => console.error('There was an error: ' + err)
       );
+    this.editApplicationForm = new FormGroup({
+      name: new FormControl('', Validators.required),
+      uuid: new FormControl('', Validators.required)
```

Comment:
Do we need the `uuid` or can we just get that from the current `application` instance?